### DISCUSSION TITLE
RWMH Logistic regression (notebook example)

### DISCUSSION THREAD

**Discussion by gerdm** [blackjax-devs/blackjax#101] [General]
Hi all!

I was wondering if you accept PR requests for example notebooks? I created a simple logistic regression example using RWMH that I think could be useful for someone who wants to see a simple example using BlackJAX.

**Comment by rlouf** [comment]
Examples are very welcome, as you can see we do not have that many yet. And logistic regression would be a great introductory example.

Feel free to open a draft PR when you start working on it and convert it to a normal PR when it is ready for review. You can also always ask for help if needed 🙂
